lib/Importer/Caffe2.cpp

```diff
@@ -49,6 +49,29 @@ static ArgumentDictionaryTy loadArgumentMap(const caffe2::OperatorDef &op) {
   return dict;
 }
 
+static std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
+  if (dict.count("pad")) {
+    int pad = loadInt(dict["pad"]);
+    std::vector<size_t> pads;
```

Comment:
You could initialize the size of the vector in the constructor or switch to std::array.
As far as I can tell this vector is not going to grow, therefore std::array is probably a better type here.
What do you think?